src/components/Icon/hooks/useIconProps.tsx

```diff
@@ -42,9 +43,11 @@ export default function useIconProps({
   }, []);
 
   const computedClassName = useMemo(() => {
-    return cx("icon_component", className, {
+    return cx("icon_component", styles.icon, className, {
       "icon_component--clickable": clickable,
-      "icon_component--no-focus-style": ignoreFocusStyle
+      [styles.clickable]: clickable,
+      "icon_component--no-focus-style": ignoreFocusStyle,
+      [styles.noFocusStyle]: ignoreFocusStyle
```

Comment:
Reverse order - first css module, then global

Reply:
I really prefer to set then every css module class next to the parallel global version and so far we did not decide on one standard

Reply:
Our one standard is modular class name first, then global one :) That's how the script is working, so it's a convention
And we've discussed at a long time ago, when I started working on the script

Reply:
Let's rediscuss

Reply:
ok i think i got it, it can be css modules before global but still every class next to it parallel right?

Reply:
Yes, sure